resources/styles/components/task-step/ends.less

```diff
@@ -40,15 +40,19 @@
     .homework-review-problem-enter-active {
       .card-body {
         .transition(~"max-height 0.5s ease-in, padding 0.5s ease-in, margin 0.5s ease-in");
-        max-height: 999px;
+        max-height: 3000px;
         padding: @tutor-card-body-padding;
```

Comment:
Have you considered switching between `transform: scaleY(0);` to `transform: scaleY(1);` ?  That avoids the `max-height` issue, but isn't quite the same effect.  https://jdsteinbach.com/css/holy-grail-css-animation/